user-interface/src/data-verification/ConsolidationOrderAccordion.tsx

```diff
@@ -55,6 +56,8 @@ export function ConsolidationOrderAccordion(props: ConsolidationOrderAccordionPr
   const [attorneys] = useState<AttorneyInfo[]>([]);
   const featureFlags = useFeatureFlags();
 
+  const api = import.meta.env['CAMS_PA11Y'] === 'true' ? MockApi : Api;
```

Comment:
Maybe we can leverage a factory pattern

Reply:
I would like to create a React hook to handle it. Basically will work just like a factory.